pkg/manifest/operations.go

```diff
@@ -297,7 +297,14 @@ func (o *Operations) uninstall(deployInfo types.InstallInfo) (bool, error) {
 	return true, err
 }
 
-func (o *Operations) getManifestForChartPath(installInfo types.InstallInfo) *types.ParsedFile {
+func (o *Operations) GetManifestForChartPath(installInfo types.InstallInfo) *types.ParsedFile {
+	if installInfo.DisableCache {
```

Comment:
Please add this to step 4. (commented) in the original getManifestForChartPath(). Since with your implementation the user provided rendered yaml will not work and will be skipped each time.
The idea is to not write a manifest.yaml to FS and if existing delete it (maybe in step 2 below).

Reply:
IMO, this is not what we want, right? The purpose of this `DisableCache` is simply make sure not generate any cached file if enabled, if no manifest.yaml write into FS upfront, why we need to go through step 1. and 2.?
And this option as we agreed, it's only for testing purpose, even if we need further logic besides this `GetRawManifest`, I prefer write another dedicated function to make sure not influence the production process logic.

Reply:
Not really. If a yaml file is provided by the user as a rendered resource manifest (not cached) in Step 1, it will never be processed. 

You can write a dedicated function no problem, but any present `manifest.yaml` should be deleted in Step 2 and not saved in Step 4.